packer/google/ubuntu/base.json

```diff
@@ -0,0 +1,94 @@
+{
+  "variables": {
+    "atlas_username":   "{{env `ATLAS_USERNAME`}}",
+    "gce_project_id":   "{{env `GCE_PROJECT_ID`}}",
+    "gce_zone":         "{{env `GCE_DEFAULT_ZONE`}}",
+    "gce_source_image": "{{env `GCE_SOURCE_IMAGE`}}",
+    "name":             "google-ubuntu-base",
+    "scripts_dir":      "scripts",
+    "config_dir":       "config",
+    "ssh_username":     "ubuntu",
+    "dns_listen_addr":  "127.0.0.1"
+  },
+  "push": {
+    "name": "{{user `atlas_username`}}/{{user `name`}}",
+    "base_dir": "../../../packer",
+    "include": [
+      "google/account.json",
+      "{{user `scripts_dir`}}/*",
+      "{{user `scripts_dir`}}/ubuntu/*",
+      "{{user `scripts_dir`}}/ubuntu/upstart/*",
+      "{{user `config_dir`}}/*",
+      "{{user `config_dir`}}/consul/*",
+      "{{user `config_dir`}}/consul_template/*",
+      "{{user `config_dir`}}/consul_template/templates/*",
+      "{{user `config_dir`}}/envconsul/*",
+      "{{user `config_dir`}}/vault/*",
+      "{{user `config_dir`}}/vault/policies/*",
+      "{{user `config_dir`}}/vault/scripts/*"
+    ],
+    "vcs": false
+  },
+  "builders": [
+    {
+      "type":              "googlecompute",
+      "project_id":        "{{user `gce_project_id`}}",
+      "account_file":      "google/account.json",
```

Comment:
Is `google/account.json` loaded from the filesystem? This should probably be an env variable for Terraform Enterprise, right?